### PR TITLE
feat: add history page

### PR DESCRIPTION
**Pull request by anik31**
## Watch History Page
I have added the Watch History page. Ignore the index & CSS files. Focus only on the components, pages & context + reducer files.

## Files to check
- src/components/Cards/HistoryCard.jsx
- src/components/Cards/VideoCard.jsx
- src/context/video-context.js
- src/pages/history/History.jsx
- src/reducer/video-reducer.js
- src/utils/serverRequests.js
 
 _PS - Ignore other files._
 
https://user-images.githubusercontent.com/56336326/160654399-16db1814-8407-49d2-bcf9-9dd643e07fed.mp4

### CODE REVIEW

File: src/utils/serverRequests.js [hunk 1]
```diff
@@ -91,3 +91,64 @@ export const removeFromWatchLaterVideos = async(id, dispatch) => {
...
+  }catch(err){
+    console.log(err);
+  }
+}
+
+export const addToWatchHistory = async(postData, dispatch) => {
+  try{
+      const {status, data} = await axios({
+        method: "post",
+        url: "/api/user/history",
```
Comment (MayankKumar10) [inline src/utils/serverRequests.js:114]: you can declare it separate variable and use that so you don't have to write it every time

File: src/components/Cards/HistoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            <i className="fas fa-trash" onClick={()=>removeFromWatchHistory(_id, dispatch)} ></i>
+            <img src={thumbnail.src} className="img-responsive" alt={thumbnail.alt} />
+            <div className="card-content">
+                <h6 className="card-title">{title}</h6>
+                <span className="text-gray text-sm">{creator}</span>
+                <div className="video-metrics">
+                    <span className="text-gray text-sm">{views} views</span>
+                    <span className="dot-separator text-gray"> • </span>
+                    <span className="text-gray text-sm">{uploaded} ago</span>
+                    {/* <div className="btn-like-dislike-wrapper">
```
Comment (MayankKumar10) [inline src/components/Cards/HistoryCard.jsx]: remove this commented code

**Review by MayankKumar10 — APPROVED** [review]
Look into these suggestions everything else looks good to me, you can merge it.

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
Everything looks good to merge. 
UI looks great

File: src/components/Cards/HistoryCard.jsx (hunk 2 above)
Comment (anik31) [reply src/components/Cards/HistoryCard.jsx]: I have removed it in the later commit

File: src/utils/serverRequests.js (hunk 1 above)
Comment (anik31) [reply src/utils/serverRequests.js:114]: Thanks for the suggestion, will try to implement it later.

**Pull request merged**